docs/devel/networking.md

```diff
@@ -271,18 +271,18 @@ take the MAC address of the first tap device attached to it,
 `preparePodNetworkInterfaces` creates a dummy nic and sets it as the first
 port of the bridge.
 
-Afterwards, the nftables / iptables rules are provisioned in the NAT table. It
+Afterwards, the nftables rules are provisioned in the NAT table. It
 follows a standard one to one NAT implementation using netfilter.
 
-It first involves the `PREROUTING` chain, which is responsible for packets that
+It first involves the `prerouting` chain, which is responsible for packets that
 have just arrived at the network interface. This rule simply filters all
 incoming traffic from the pod networking interface, making it go through the
 `KUBEVIRT_PREINBOUND` chain.
 
 ```
-Chain PREROUTING (policy ACCEPT)
-target               prot opt in   source               destination
-KUBEVIRT_PREINBOUND  all  --  eth0 anywhere             anywhere
+chain prerouting {
+		iifname "eth0" counter jump KUBEVIRT_PREINBOUND
```

Comment:
please make this consistent with the other rules and use less spaces to indent it.